source/actions3/actions/AMControlMoveAction3.cpp

```diff
@@ -151,6 +151,7 @@ void AMControlMoveAction3::onMoveReTargetted()
 
 void AMControlMoveAction3::onMoveFailed(int reason)
 {
+	bool reportMoveError = true;
```

Comment:
This bool is unnecessary because you can just compare against the enum failure value itself.  I'll go ahead and change it since @XuetaoLiu-CLS is away.